app/src/app/components/ContextMenu.tsx

```diff
@@ -22,7 +28,48 @@ export const MapContextMenu: React.FC = () => {
       console.log("Error updating assignments: ", error);
     },
     onSuccess: (data) => {
```

Comment:
@nofurtherinformation implementing this `onSuccess` is where I left off, if you're looking for a place to pick up

Reply:
Note that setting the parent feature state and filter doesn't work properly. 

Additionally, we'll need to remove the shattered parents from the map store so they don't get added back in on the next paint event. That's honestly good enough reason for me not to track ALL of the assignments in the map store–which we presently do–instead just store a buffer of unsaved assignments (which should get cleared after successfully sending them to the db).

Reply:
Great. My approach here was to store the IDs of the shattered parents and children in the mapStore, and subscribe to that whenever a shatter event occurs. Then, use the `in` [filter on the parent and child layers](https://github.com/districtr/districtr-v2/pull/84/files#diff-4030c7e740e0aa657978a3a23f1e6eada92c0389829283ca4a50ffbf1f774b9bR215-R225). This could be swapped out for setFeatureState. 

There are still interactions to handle and reloading the shatters when the user returns, but, so far: 

<img width="1056" alt="Screenshot 2024-09-24 at 4 36 59 PM" src="https://github.com/user-attachments/assets/484228d8-ccec-4a52-a744-8fae469f6720">
<img width="1056" alt="Screenshot 2024-09-24 at 4 37 02 PM" src="https://github.com/user-attachments/assets/2e36e4cb-4497-4855-97cc-c95669eb815f">